.pre-commit-search-and-replace.yaml

```diff
@@ -0,0 +1,6 @@
+- search: from astropy import units as u
+  replacement: import astropy.units as u
+- search: from astropy import constants as const
+  replacement: import astropy.constants as const
+- search: Plasmapy|PlasmPy|Plasmpy|Palsmapy
+  replacement: PlasmaPy
```

Comment:
...to catch common typos that have snuck in before.

Reply:
Good Idea 👍